AdobeUi/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "magento/module-ui",
+    "description": "N/A",
+    "require": {
+        "php": "~7.1.3||~7.2.0",
+        "magento/framework": "*",
+        "magento/module-adobe-stock-asset-api": "*",
```

Comment:
AdobeUi module is an extraction of masonry grid and overlay that will be delivered to Magento core. It should not depend on other adobestock modules